remote_config/src/desktop/remote_config_request.h

```diff
@@ -21,12 +21,20 @@
 #include "remote_config/request_generated.h"
 #include "remote_config/request_resource.h"
 
+#ifdef FIREBASE_TESTING
+#include "testing/base/public/gunit.h"
+#endif  // FIREBASE_TESTING
+
```

Comment:
Intrigued. I don't know if I've seent his before.  Is this a standard practice?

Reply:
So first this code change is not correct, I was using the reference from internal pattern so I changed that. Second I don't know whether it's a standard practice, but we do have existing case in rest.h, I guess we can accept this?